celestia/peers.txt

```diff
@@ -16,3 +16,4 @@ d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
 a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
 2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
 5001de72be39622c9dc34f2117eccc3f3fca8a7a@celestia-mainnet-01.twinstake.io:26756
+e92c1f9c189018d4b94899f99ee24d0b90f6b9b5@217.160.39.214:26656
```

Comment:
Can you do this instead:
Please make the peers and gentx separate files, so that your PR looks like this:

celestia/gentx/validatorname.json // 
celestia/validatorname.txt //